app/.prettierignore

```diff
@@ -4,4 +4,5 @@
 /test/unit/tmp/
 /docs/cosmos-voyager/
 /**/builds/*
-/.vscode/*
\ No newline at end of file
+/.vscode/*
+../common/
```

Comment:
?

Reply:
that is the location for the starname file. Prettier was throwing lots of warning for semicolon

Reply:
let's fix the warnings instead

Reply:
ok, fixed